tests/features/joinup_subscription/subscriptions_manage.feature

```diff
@@ -2,12 +2,110 @@
 Feature: User subscription settings
   As a user I must be able to set and view my subscription settings.
 
-  Scenario Outline: Change the notification frequency of my digests
+  Background:
     Given user:
       | Username   | Auric Goldfinger  |
       | Password   | oddjob            |
       | E-mail     | auric@example.com |
-    And collection:
+
+  Scenario: Check access to the subscription management pages
+    Given user:
+      | Username   | Chanelle Testa    |
+      | E-mail     | chate@example.com |
+
+    # No access for anonymous users.
+    Given I am an anonymous user
+    When I go to the subscription settings form of "Auric Goldfinger"
```

Comment:
I agree with this, and I have already provided what you request in the functional test. Here are the steps that fulfil this:

```
    Given I am logged in as "Auric Goldfinger"
    And I am on the homepage
    When I click "My account"
    And I click "Subscription settings" in the "Header" region
    Then I should see the heading "Subscription settings"
```

I think it is a good practice to provide a step definition such as `When I go to the subscription settings form` so it can be used in secondary tests like access tests. For example when testing that an anonymous user doesn't have access to the page we shouldn't really follow all the steps to then notice that the contextual link is not present. It is quicker and more correct to actually navigate to the page itself and verify it throws a 304.